mcs/class/System.Private.CoreLib/System.Diagnostics/StackTrace.cs

```diff
@@ -1,20 +1,54 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
 namespace System.Diagnostics
 {
+	// Need our own stackframe class since the shared version has its own fields
+	[StructLayout (LayoutKind.Sequential)]
+	class MonoStackFrame {
+		#region Keep in sync with object-internals.h
+		internal int ilOffset;
+		internal int nativeOffset;
+		internal long methodAddress;
```

Comment:
could you add comment on fields which are unused

Reply:
Fixed.